android/fastlane/Fastfile.private

```diff
@@ -0,0 +1,87 @@
+platform :android do
+
+  desc "Validate"
+  private_lane :validate do |options|
+    # TODO: If the release isn't production, 
+    # validate if there is a user logged on firebase
+    UI.message "Validating..."
+  end
+
+  desc "Builds the app creating the `.apk` or `.aab` files"
+  private_lane :build_application do |options|
+    environment = options[:environment]
+    is_firebase = options[:firebase]
```

Comment:
I'd change this to `aab`, wich is android app bundle, and I revert their value, so only google play release have this boolean on true